penrose-web/src/engine/Evaluator.ts

```diff
@@ -355,15 +468,22 @@ export const evalExpr = (
           throw Error(`expected 1 argument to ${fnName}; got ${argExprs.length}`);
         }
 
-        if (argExprs[0].tag !== "EPath") {
-          throw Error(`expected 1 path as argument to ${fnName}; got ${argExprs[0].tag}`);
```

Comment:
This is a very special case for these two functions. Maybe separate the body out to a function?

Reply:
I think it's okay to leave this inline since there would be no other callers to the function if I separated it out.